dotCMS/src/main/webapp/html/portlet/ext/cmsmaintenance/tail_log.jsp

```diff
@@ -830,6 +830,13 @@
             logViewManager.updateView(data);
         });
 
+        sseSource.addEventListener('keepAlive', function(e) {
+
+            // Assuming we receive JSON-encoded data payloads:
+            const data = JSON.parse(e.data);
+            console.log("keepAlive :: " + data.keepAlive);
```

Comment:
does this log consume too much resources from the browser if the idle time if too long?